apiary-metastore-auth/README.md

```diff
@@ -0,0 +1,20 @@
+# Apiary ReadOnlyAuth pre-event listener
+
+## Overview
+The ReadOnlyAuth pre-event listener is an optional Apiary component which, if enabled, will handle authorization using static database list.
+
+## Installation
+The listener can be activated by placing its jar file on the Hive metastore classpath and configuring Hive accordingly. For Apiary 
+this is done in [apiary-metastore-docker](https://github.com/ExpediaInc/apiary-metastore-docker). 
+
+## Configuration
+The ReadOnly pre-event listener can be configured by setting the following System Environment variables:
+
+|Environment Variable|Required|Description|
+|----|----|----|
+SHARED_HIVE_DB_NAMES|Yes|Comma-separated database names to allow access.
```

Comment:
Could we rename this to READABLE_DATABASE_WHITELIST so at least it's a bit more consistent with the terminology in other Apiary components like WaggleDance?

Reply:
I was following pattern on managed databases, which has variable HIVE_DB_NAMES. If you think this is more appropriate I can change, we also need to update terraform code to match the change.
https://github.com/ExpediaInc/apiary-data-lake/blob/master/templates/apiary-hms-readonly.json#L49

Reply:
Hmm, OK, the problem is that the word "SHARED" can mean many different things while I think it's more accurate to consider this a whitelist right? How about "HIVE_DB_WHITELIST" or "HIVE_DB_NAMES_WHITELIST" as alternatives?